MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -45,6 +45,20 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      */
     @objc public var abbreviationPriority: Int = NSNotFound
     
+    /**
+     An array indicating which directions you can go from a lane (left, right, or straight).
+     
+     If the value is `[.left", .straight]`, the driver can go left or straight from that lane. This is only set when the `component` is a `lane`.
+     */
+    @objc public var indications: LaneIndication
+    
+    /**
+     The boolean that indicates whether the component is a lane and can be used to complete the upcoming maneuver.
+     
+     If multiple lanes are active, then they can all be used to complete the upcoming maneuver. This value is set to `false` by default.
+     */
+    @objc public var isLaneActive: Bool = false
```

Comment:
I think we should strive for more similarities with how lane indications are normally exposed. It’s poor API design to expose multiple mutually exclusive properties that are optionally set depending on each other. Consider making VisualInstructionComponent an umbrella class or protocol for subclasses such as LaneInstructionComponent – `isActive` can then be a property on that subclass.

Reply:
Better yet, instead of a bunch of lane instruction components, build up a new `Intersection` object that can have a bunch of `approachLanes` and `usableApproachLanes`.